src/index.ts

```diff
@@ -2,6 +2,8 @@ import { randomBytes, createHmac } from "crypto"
 
 const charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ234567"
 
+export type hashAlgorithm = "SHA1" | "SHA256";
```

Comment:
Should we also support SHA512? See https://www.rfc-editor.org/rfc/rfc6238#appendix-A.

Reply:
Added